lightbend-telemetry/log-correlation-java-mvn/.gitignore

```diff
@@ -0,0 +1,16 @@
+target
+ideatarget
+.target
+bin
+.cache
+.cache-main
+.cache-tests
+.classpath
+.project
+.tmpBin
+.factorypath
+.settings
+logs
+.idea
+*.iml
+RUNNING_PID
```

Comment:
do we need the per-folder `.gitiignore`?
These changes look valuable at root-level. I'm +1 to moving them to `/.gitignore`.

Reply:
Fixed